.github/workflows/cd-syft-dev.yml

```diff
@@ -150,3 +162,24 @@ jobs:
           add: "."
           push: "origin main"
           cwd: "./infrastructure/"
+
+      - name: Cleanup Azure Container Registry
+        uses: azure/CLI@v1
+        with:
+          # SKIP_LINES = latest version dev & dev-<sha> (2 lines) + keep "n" previous version (n lines) + 1
+          inlineScript: |
+            ACR_REGISTRY_NAME=${{ secrets.ACR_REGISTRY_NAME }}
+            REPO_LIST=$(az acr repository list -n $ACR_REGISTRY_NAME -o tsv)
```

Comment:
Q: Do we not need to provide azure credentials to access the private registry?

Reply:
I see it failed because I made a mistake - there's a difference between logging into the container registry vs. logging into azure. Will add the **azure** login step.